apps/root/src/services/earnService.spec.ts

```diff
@@ -714,13 +716,118 @@ describe('Earn Service', () => {
                   // @ts-expect-error do not care
                   token: createSdkTokenWithWithdrawTypesMock({}),
                   amount: '500000000000000000',
+                  withdrawType: WithdrawType.IMMEDIATE,
                 },
               ],
               positionId: '10-0xvault-30',
               strategyId: '0xvault' as `${number}-${Lowercase<string>}-${number}`,
             } satisfies EarnWithdrawTypeData['typeData'],
           },
         },
+        {
+          expectedPositionChanges: {
```

Comment:
can we also add a case for one that did not have delayed before?

Reply:
I think this was the case, but I have added one that had some delayed before, so checks the sum :)